pkg/apis/hco/v1beta1/zz_generated.openapi.go

```diff
@@ -1,8 +1,7 @@
-//go:build !ignore_autogenerated
```

Comment:
please also add `go fmt` because it adds the `//go:build !ignore_autogenerated` later, and it will fail in sanity

Reply:
done